src/components/materialization/Create.tsx

```diff
@@ -85,16 +87,6 @@ function MaterializationCreate() {
                 setFailureState();
             }
         },
-        doneSubscribing: (subscription: RealtimeSubscription) => {
-            return supabaseClient
-                .removeSubscription(subscription)
-                .then(() => {
-                    setFormState({
-                        status: FormStatus.IDLE,
-                    });
-                })
-                .catch(() => {});
-        },
```

Comment:
These helpers weren't needed to be stand alone because only the function above used them.